person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission || is_of_age?
```

Comment:
- [OPTIONAL] Good job @AlierPM, I think it would be better if you correct some typos. The method you defined is `of_age` but here you write `is_of_age`. Please kindly fix that 🙃 😉